src/tests/ftest/data/daos_server_baseline.yaml

```diff
@@ -19,7 +19,7 @@ servers:
 -
   targets: 8                # count of VOS targets
   first_core: 0             # starting index for targets
-  nr_xs_helpers: 2          # offload helpers per target
+  nr_xs_helpers: 2          # offload helpers per server
```

Comment:
suggest this be set to 16 consistent with other review comments

Reply:
OK, just refreshed and changed the cases you pointed out, thx.